src/Revit/RevitHyparTools/WallByProfile.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Elements;
+using Elements.Geometry;
+using Elements.Geometry.Solids;
+
+namespace RevitHyparTools {
+    [UserElement]
+    public class WallByProfile : Wall
```

Comment:
Always add `///` comments to the class and to its methods and fields.

Reply:
this is moving to be a generated type, so I'm not going to do this.